starknet_py/net/gateway_schemas/gateway_schemas.py

```diff
@@ -114,8 +114,12 @@ class TransactionReceiptSchema(Schema):
     block_number = fields.Integer(data_key="block_number", load_default=None)
     version = fields.Integer(data_key="version", allow_none=True)
     actual_fee = Felt(data_key="actual_key", allow_none=True)
-    rejection_reason = fields.String(
-        data_key="transaction_rejection_reason", allow_none=True, load_default=None
```

Comment:
Just acknowledging that the data_key was incorrect in this case and since our tests didn't check error messages we didn't catch that it's not working correctly.

Reply:
IIR I encountered a validation error because it's effectively a `dict`.